src/Tizen.NET.MaterialComponents/Components/MButton.cs

```diff
@@ -71,11 +71,9 @@ public override Color BackgroundColor
             set
             {
                 base.BackgroundColor = value;
-
-                var pressedColor = value.WithAlpha(0.32);
                 var disabledColor = value.WithAlpha(0.12);
 
-                SetPartColor(Parts.Widget.BackgroundPressed, pressedColor);
+                SetPartColor(Parts.Widget.BackgroundPressed, value);
```

Comment:
any special reason do this?

Reply:
No. I removed the line.

Reply:
Is there no reason removing this? As far as I know, `pressed` color is different with background color.